src/index.js

```diff
@@ -0,0 +1,33 @@
+import './style.css';
+
+const todosListEl = document.getElementById('todos-list');
+
+const todos = [
+  {
+    description: 'task1',
+    isCompleted: true,
+  },
+  {
+    description: 'task2',
+    isCompleted: false,
+  },
+  {
+    description: 'task3',
+    isCompleted: false,
+  },
```

Comment:
- [ ] This should be `completed`, not `isCompleted`.

- [ ] You are missing the `index` key also.

  Each task should be like this:

  ```
  {
    description: 'task 1',
    completed: true,
    index: 1
  }
  ```